src/musicbrainz/web/musicbrainzrecordingstask.cpp

```diff
@@ -161,11 +177,31 @@ void MusicBrainzRecordingsTask::doNetworkReplyFinished(
         auto trackReleases = m_trackReleases.values();
         m_trackReleases.clear();
         emitSucceeded(trackReleases);
-        return;
+        return true;
     }
 
     // Continue with next recording id
     DEBUG_ASSERT(!m_queuedRecordingIds.isEmpty());
+    emit currentRecordingFetched();
+    auto inhibitTimerElapsed = m_measurementTimer.elapsed(true);
+    qDebug() << "Task took:" << inhibitTimerElapsed.toIntegerMillis();
+    qDebug() << kMinTimeBetweenMbRequests;
+    m_requestTimer.setSingleShot(true);
+    if (inhibitTimerElapsed.toIntegerMillis() >= kMinTimeBetweenMbRequests) {
```

Comment:
toIntegerMillis() can be moved to above for deduplication

Reply:
I tried something, I hope that is correct.

Reply:
Did you forget to push?

Reply:
Oh yeah I'm so sorry. I'm pushing now :+1: